### PR TITLE
feature/GEINFRA-236_GEINFRA-249: Update wizard cta logic

### PR DESCRIPTION
**Pull request by AltusBarry**

### CODE REVIEW

File: authentication_service/templates/formtools/wizard/wizard_form.html [hunk 1]
```diff
@@ -15,14 +15,27 @@
...
-        </button>
-        <button class="Button" name="wizard_goto_step" type="submit" value="{{ wizard.steps.prev }}">
-            {% trans "Prev step" %}
-        </button>
-    {% endif %}
-    <input class="Button"type="submit" name="submit" value="{% trans "Submit" %}" />
+    {% with current_step=wizard.steps.current prev_step=wizard.steps.prev first_step=wizard.steps.first last_step=wizard.steps.last%}
+        {# Submit button text should display next for all steps except the last #}
+        {% if current_step == last_step %}
+            <input class="Button"type="submit" name="submit" value="{% trans "Submit" %}" />
```
Comment (cobusc) [inline authentication_service/templates/formtools/wizard/wizard_form.html]: Space between "Button" and `type`. Is double quote inside double quote working? `value="{% trans "Submit" %}"`

File: authentication_service/templates/formtools/wizard/wizard_form.html [hunk 2]
```diff
@@ -15,14 +15,27 @@
...
+        {% else %}
+            <input class="Button"type="submit" name="submit" value="{% trans "Next" %}" />
```
Comment (cobusc) [inline authentication_service/templates/formtools/wizard/wizard_form.html]: Space between "Button" and `type`. Is double quote inside double quote working? `value="{% trans "Submit" %}"`

File: authentication_service/tests/test_views.py [hunk 3]
```diff
@@ -1184,6 +1184,12 @@ def test_incorrect_required_field_logger(self):
             'WARNING:authentication_service.forms:Received required field that is not on form: notontheform',
             'WARNING:authentication_service.forms:Received required field that is not on form: notontheform',
             'WARNING:authentication_service.forms:Received required field that is not on form: notontheform',
+            'WARNING:authentication_service.forms:Received required field that is not on form: notontheform',
```
Comment (cobusc) [inline authentication_service/tests/test_views.py]: Remember my `"Foo" * 7` proposal from a previous PR? It is still not to late to use it.

**Comment by cobusc** [comment]
Left some questions/suggestions, but :+1: in general

File: authentication_service/tests/test_views.py (hunk 3 above)
Comment (AltusBarry) [reply authentication_service/tests/test_views.py]: It needs to hurt when we see it. Make us understand what happens when we wizard :stuck_out_tongue:

File: authentication_service/templates/formtools/wizard/wizard_form.html (hunk 1 above)
Comment (cobusc) [reply authentication_service/templates/formtools/wizard/wizard_form.html]: You'll need to convince Github, not me. They render it in bright red...like two ominous eyes staring at me. :)

**Pull request closed without merging**